.github/workflows/changelog.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     branches:
       - master
-      - zowe-v1-lts

```

Comment:
there may be other workflows in this repo that could use the same update.
but I don't think it particularly matters for this branch or until we create a zowe-v2-lts branch